src/components/Header/utils/types.tsx

```diff
@@ -0,0 +1,19 @@
+export enum HeaderType { Link, Anchor }
```

Comment:
This should be a `ts` file since no React is in use here :)

Reply:
types shouldn't be under utils folder. 
I think it makes sense just to be on Header level